bigframes/bigquery/__init__.py

```diff
@@ -18,6 +18,7 @@
 
 import sys
 
+from bigframes.bigquery._operations.ai import ai_generate_bool
```

Comment:
To match SQL better, let's actually do this:

```suggestion
from bigframes.bigquery._operations import ai import
```